std/algorithm/setops.d

```diff
@@ -3,6 +3,18 @@
 This is a submodule of $(MREF std, algorithm).
 It contains generic algorithms that implement set operations.
 
+The functions : $(LREF multiwayMerge), $(LREF multiwayUnion),
+$(LREF setDifference), $(LREF setIntersection), $(LREF setSymmetricDifference)
+expect a range of sorted ranges as input; providing a range of unsorted
+ranges will lead to a behaviour which is independent with the specification.
+
+The algorithms implemented specifically for sets : $(LREF setDifference),
+$(LREF setIntersection), $(LREF setSymmetricDifference) are not limited at
+working only with sets, but also with multisets. The rule of thumb is the following:
+if the input contains only sets, the result is going to be a set, otherwise
```

Comment:
How about being precise here? Either all algorithms behave like this or it makes sense to make a short table.